Lib/ufo2ft/outlineCompiler.py

```diff
@@ -1407,7 +1408,7 @@ def compileGlyphs(self):
                 logger.error("%r has invalid curve format; skipped", name)
                 ttGlyph = Glyph()
             else:
-                ttGlyph = pen.glyph()
+                ttGlyph = pen.glyph(componentFlags=0x0)
```

Comment:
Wouldn't this change composite glyph flags of all the fonts including unhinted ones? Up until now, we would have ROUND_XY_TO_GRID set for all components by default. Why is the change needed here? Where was it discussed?

Reply:
This was discussed in #425 but concluded to not change.

Reply:
That line doesn’t actually change the result, it just starts with flags set to 0x0. The ROUND_XY_TO_GRID flag is still set by default on all components by https://github.com/googlefonts/ufo2ft/blob/6ef8e92b51120fe9bd467fbc76bd3aac979b6241/Lib/ufo2ft/instructionCompiler.py#L164-L170 respectively https://github.com/googlefonts/ufo2ft/blob/6ef8e92b51120fe9bd467fbc76bd3aac979b6241/Lib/ufo2ft/instructionCompiler.py#L182-L189

It just seemed weird to set one particular flag at a completely different point in the code than the others.